static/skin/index.js

```diff
@@ -5,13 +5,15 @@
         count: viewPortToCount()
     };
     const filterTypes = ['lang', 'category', 'q'];
-    const bookMap = new Map();
+    const bookOrderMap = new Map();
```

Comment:
Why rename this to `bookOrderMap` ?

Are the books ordered ?

Reply:
@mgautierfr yes the books are ordered and this is something that was requested by veloman in my previous PR but got missed so adding it here.

Reply:
Haa, I've missed that the map is not a map between a book and a id but is a map between a book id and the order of the book.
We are good. (But it would have been nice to have this in a separated commit. It has nothing to do with "added cookies for filter effect". And the commit message could have explain why we've changed this.)